app/controllers/account/admin/companies_imports_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Account::Admin::CompaniesImportsController < Account::Admin::AdminController
+  def new
+    @companies_import = CompaniesImport.new
+  end
+
+  def create
+    @companies_import = CompaniesImport.new(params[:companies_import])
+    if @companies_import.save
+      flash[:success] = "Companies were imported."
+      redirect_to account_admin_companies_path
+    else
+      render :new
+    end
+  rescue StandardError => e
```

Comment:
you should collect import errors not rescue an exception

Reply:
Is it good?
![Screenshot from 2020-10-30 09-44-50](https://user-images.githubusercontent.com/31774981/97673555-a58e9900-1a94-11eb-8aa2-5852d01c34a7.png)